examples/agents_use_cases/agent_financial.py

```diff
@@ -5,8 +5,7 @@
 from dynamiq.utils.logger import logger
 from examples.llm_setup import setup_llm
 
-AGENT_ROLE = "A helpful and general-purpose AI assistant that has strong language skills, Python skills, and Linux command line skills."  # noqa: E501
-AGENT_GOAL = """is to provide concise answer to user,
+AGENT_ROLE = """A helpful and general-purpose AI assistant that has strong language skills, Python skills, and Linux command line skills.Goal is to provide concise answer to user,
```

Comment:
Please double check format of this string. I see a lot of spaces, not sure if this expected